packages/data-point/lib/entity-types/entity-request/resolve.js

```diff
@@ -91,25 +92,58 @@ function resolveUrl (acc) {
 module.exports.resolveUrl = resolveUrl
 
 /**
+ * @param {Function} callback
+ * @returns {Function} a new callback, that calls the original one
+ */
+function getRequestPromiseWithDebugging (callback) {
+  const debugModule = require('request-debug')
+  clearModule('request-promise')
```

Comment:
If we require `request-promise`, make changes to the object, and then require it again, there's a chance that the second require will also get the modified version of the module. Clearing it a second time is a way of trying to avoid this problem.